### ISSUE TITLE
Consider actual return code

### ISSUE THREAD

**Issue by stakemura** [awni/cuthon#3]
I guess it would be better to consider the actual return code on success and change return code to non-zero value on failure.

I would be grateful if you would modify it like the following code.

```
    ret = subprocess.call(" ".join(cmd_args), env=os.environ, shell=True)
    sys.exit(ret)
```
```
    except OSError as e:
        print("nvidia-smi not found, aborting.")
        sys.exit(1)
```
```
    if len(gpus) < args.num_gpus:
        print("Not enough free GPUs found, aborting.")
        sys.exit(2)
```

**Comment by awni** [comment]
Good suggestion! If you are interested in contributing, please submit a PR and I will merge. O/w I will add this when I have a few minutes.

**Comment by awni** [comment]
Done with commit a7d26d22a1f6fefaecedbf9065a2cd6b0b4ad88e and released to pypi (v0.5)
